Update EIP-3561: Move from Stagnant to Review

SamPorter1984 opened the pull request:
Shorter, clearer. Slightly better rationale. Updated implementation link.

Pandapip1 reviewed (dismissed):
Please update the status to review and fix the EIP-Bot errors.

SamWilsn reviewed (changes requested):
This EIP is in the Stagnant state, so you'll have to move it back into an active state before you can make changes.

Also, please see the errors in the Checks section. You'll need to fix those too.

Thanks!

SamPorter1984 commented:
Bot checks passed
And uh, fulfilled your requests
Also, changed the specification a bit, moved away from trust minimized boolean, to a more concise and logical zero trust period, which effectively makes the code multi-chain.
If ever etherscan and it's forks will not only check proxy slots, but also code validity of a proxy, this one is ready.
